src/_tests/integration/cuj2.integration-test.js

```diff
@@ -0,0 +1,34 @@
+const { click, findText, waitForNoSpinners, signIntoTerra } = require('./integration-utils')
```

Comment:
I think we should avoid references to 'CUJ' in the code, since that name may not have a long life span. How about naming the file e.g. `find-workflow.integration-test.js`?